.golangci.yml

```diff
@@ -2,83 +2,74 @@
 # SPDX-License-Identifier: MPL-2.0
 
 run:
-  # timeout for analysis, e.g. 30s, 5m, default is 1m
+  # Timeout for analysis
   deadline: 10m
 
-  tests: false
+  # Include test files
+  tests: true
 
-  # which dirs to skip: they won't be analyzed;
-  # can use regexp here: generated.*, regexp is applied on full path;
-  # default value is empty list, but next dirs are always skipped independently
-  # from this option's value:
-  #   	vendor$, third_party$, testdata$, examples$, Godeps$, builtin$
+  # Skip directories
   skip-dirs:
     - iopodman
     - build
+    - examples
+    - scripts
 
-  # which files to skip: they will be analyzed, but issues from them
-  # won't be reported. Default value is empty list, but there is
-  # no need to include all autogenerated files, we confidently recognize
-  # autogenerated files. If it's not please let us know.
+  # Skip files
   skip-files:
     - ".*\\.generated\\.go$"
     - ".*bindata_assetfs\\.go$"
```

Comment:
I think these are copy-pasta from Nomad and could be removed as well. No harm in keeping them thought 🤷